workers/limiter/src/index.ts

```diff
@@ -160,6 +156,19 @@ export default class LimiterWorker extends Worker {
     this.logger.info('Limiter worker started regular check');
 
     const report = await this.analyzeWorkspacesLimits();
+    
+    report.bannedProjectIds.map(async (projectId) => {
+      /**
+       * If project is not in the set now, it would be banned
+       */
+      if (!this.redis.isProjectBanned(projectId)) {
+        const project = await this.projectsCollection.findOne({
+          '_id': projectId,
+        });
+
+        telegram.sendMessage(`Block project ${project.name}`, telegram.TelegramBotURLs.Limiter);
```

Comment:
Use past-tense in logs. And maybe logs should be after the project is actually banned?